universal-application-tool-0.0.1/app/services/program/ExportDefinition.java

```diff
@@ -0,0 +1,35 @@
+package services.program;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.google.auto.value.AutoValue;
+import javax.annotation.Nullable;
+
+@AutoValue
+@JsonDeserialize(builder = AutoValue_ExportDefinition.Builder.class)
+public abstract class ExportDefinition {
+  @JsonProperty("engine")
+  /** Which export engine to use. */
+  public abstract String engine();
+
+  @JsonProperty("pdfConfig")
+  @Nullable
+  /** The configuration of the PDF export - only if engine == "pdf". */
+  public abstract PdfExportConfig pdfConfig();
+
+  @JsonProperty("csvConfig")
+  @Nullable
+  /** The configuration of the CSV export - only if engine == "csv". */
```

Comment:
nit: comments should be above the annotations since the annotation is part of thing being commented

Reply:
Done.